alohomora/main.py

```diff
@@ -238,8 +240,16 @@ def main(self):
                 role_arn = selectedrole.split(',')[0]
                 principal_arn = selectedrole.split(',')[1]
 
-        token = alohomora.keys.get(role_arn, principal_arn, assertion, duration)
-        alohomora.keys.save(token, profile=self._get_config('aws-profile', DEFAULT_AWS_PROFILE))
+        try:
+            token = alohomora.keys.get(role_arn, principal_arn, assertion, duration)
+            alohomora.keys.save(token, profile=self._get_config('aws-profile', DEFAULT_AWS_PROFILE))
+        except botocore.exceptions.ClientError as err:
+            details = err.response['Error']
+            if details['Code'] == 'ValidationError' and \
+               details['Message'].find('DurationSeconds exceeds the MaxSessionDuration') != -1:
+               print(details['Message'])
+               sys.exit(0)
```

Comment:
Might be better if we exit with a non-zero status.

Reply:
I looked at what was used for other error conditions. Not saying it is correct, but it's consistent.